custom-namespaces.yaml

```diff
@@ -342,49 +342,16 @@ firefox_accounts:
     - loines@mozilla.com
   pretty_name: Firefox Accounts
   views:
-    all_events:
+    events:
```

Comment:
According the data catalog there are [154 Looker charts using the `firefox_accounts.event_counts` explore](https://mozilla.acryl.io/dataset/urn:li:dataset:(urn:li:dataPlatform:looker,firefox_accounts.explore.event_counts,PROD)/Lineage), which is being changed here to select from `fxa_all_events` rather than `fxa_content_auth_oauth_events`.  That means those Looker charts will now be including the `stdout` events as well, which may not be what the people who created the charts want to happen.